docs/rfc/2022-07-05-mo-cluster.md

```diff
@@ -87,6 +87,44 @@ It is worth noting that `MatrixOneCluster` object is not necessary to create a m
 - advanced user can ignore `MatrixOneCluster` object perform fine-grained control over `*Set` directly;
 - it is trivial to support heterogenous cluster: one simply deploy several `*Set` objects with different Pod specification;
 
+An overview of the deployment topology:
+
+```mermaid
+flowchart TD
+  subgraph k8s
+    subgraph Cluster
+    direction TB
+    LoadBalancer
+    subgraph CN
```

Comment:
CNs need to communicate with each other, as well as DNs

Reply:
Good catch

Reply:
better?